.gitignore

```diff
@@ -1,3 +1,4 @@
 .dub
 build
 dub.selections.json
+test/generated
```

Comment:
Please add the missing new line at the end of file

Reply:
Done